### PR TITLE
Add Listen and Watch translations

### LINKED ISSUE #4863: Add translations for watch and listen in service configs
**Is your feature request related to a problem? Please describe.**
There are no translations of `Watch` and `Listen` in the service configs. They are in english instead of translations.

**Describe the solution you'd like**
Add translations in the service configs. Check with the translations spreed sheet.

**Describe alternatives you've considered**

**Testing notes**
- Manually test that each new translation is as expected in the bulletin component for that service - screen reader text and the call to action.
- Check with editorial that these are correct.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #4863

**Overall change:** 
Add translations for `Listen` and `Watch` in service configs.

**Code changes:**
- Add `Listen` and `Watch` translations in the services which have a translation in the spreadsheet.
- Update tests
- Update snapshots
- Add service knob to stories

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/Bulletin/index.stories.jsx [hunk 1]
```diff
@@ -33,33 +35,50 @@ const audioFixture = bulletinFixture('RadioBulletin');
 const liveTvFixture = liveBulletinFixture('TVBulletin');
 const audioLiveFixture = liveBulletinFixture('RadioBulletin');
 
-const getBulletinPromo = platform => item => (
-  <ServiceContextProvider service="news">
+const getBulletinPromo = (platform, service, item) => (
+  <ServiceContextProvider service={service}>
     <RequestContextProvider
```
Comment (OlgaLyubin) [inline src/app/containers/Bulletin/index.stories.jsx:40]: Everything looks good, translations match the spreadsheet. The only thing is that storybook gives a warning for Bulletin : `Warning: Failed prop type: The prop pathname is marked as required in RequestContextProvider, but its value is undefined`.

File: src/app/containers/Bulletin/index.stories.jsx [hunk 2]
```diff
@@ -33,33 +35,50 @@ const audioFixture = bulletinFixture('RadioBulletin');
...
       bbcOrigin="https://www.test.bbc.co.uk"
       isAmp={platform === 'amp'}
       pageType="frontPage"
-      service="news"
+      service={service}
```
Comment (Bopchy) [inline src/app/containers/Bulletin/index.stories.jsx:45]: `pathname` needs to be defined, as the lack of it is causing this console warning `Warning: Failed prop type: The prop pathname is marked as required in 'RequestContextProvider', but its value is 'undefined'.`

File: src/app/containers/Bulletin/index.stories.jsx (hunk 1 above)
Comment (DenisHdz) [reply src/app/containers/Bulletin/index.stories.jsx:40]: Added also to the `StoryPromo` stories as it was also missing and logging a warning.

File: src/app/containers/Bulletin/index.stories.jsx (hunk 2 above)
Comment (Bopchy) [reply src/app/containers/Bulletin/index.stories.jsx:45]: Sorry, hadn't refreshed my page to see the previous comment

**Review by OlgaLyubin — APPROVED** [review]
Looks good to me

**Review by Bopchy — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
Currently, `serbian-lat` is using the same translations as `serbian-cyr`. This is incorrect. I've spoken with @DenisHernandez regarding this and he's going to update the config so this no longer happens.

**Comment by DenisHdz** [comment]
> Currently, `serbian-lat` is using the same translations as `serbian-cyr`. This is incorrect. I've spoken with @DenisHernandez regarding this and he's going to update the config so this no longer happens.

We have decided to keep the serbian translations in latin in this PR, and we will split them into the two variants in a separated PR.

**Comment by jamesbrumpton** [comment]
lgtm

**Pull request merged**